### PR TITLE
Use auth-api param

### PR DESCRIPTION
**Pull request by JamesJeffryes**
I suspect that using auth-service-url previously was causing problems

### CODE REVIEW

File: deployment/conf/.templates/deploy.cfg.templ [hunk 1]
```diff
@@ -14,6 +14,8 @@ mongodb-pwd = {{ default .Env.mongodb_pwd "" }}
 
 # The KBase auth server url.
 auth-service-url = {{ default .Env.auth_service_url "https://kbase.us/services/authorization/Sessions/Login" }}
+# The KBase auth API.
+auth-service-api = {{ default .Env.auth_service_url "https://ci.kbase.us/services/auth" }}
```
Comment (bio-boris) [inline deployment/conf/.templates/deploy.cfg.templ]: It's currently set to auth_service_url=https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login

File: lib/biokbase/catalog/controller.py [hunk 2]
```diff
@@ -1166,7 +1166,7 @@ def has_permission(self, username, token, owners):
         return False
 
     def is_admin(self, username, token):
-        r = requests.get(self.auth_url + '/api/V2/me', headers={'Authorization': token})
+        r = requests.get(self.auth_api + '/api/V2/me', headers={'Authorization': token})
```
Comment (bio-boris) [inline lib/biokbase/catalog/controller.py]: It's currently set to 'auth_service_url=https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login'

Comment (bio-boris) [reply lib/biokbase/catalog/controller.py]: So this won't work.

**Comment by JamesJeffryes** [comment]
@bio-boris If you are OK with this. Let's merge into develop. Thanks for the help!

**Comment by bio-boris** [comment]
@JamesJeffryes Are you sure the is_admin is working?

**Pull request merged**